cloud/passportauth/src/user/UserRepository.ts

```diff
@@ -9,9 +7,10 @@ export interface UserRepository {
    * Retrieves a user object from a data source by login id (i.e. username, email)
    *
    * @param loginId - A unique login id used to identify the user
+   * @param cb - Callback
    * @returns {Promise<any>} - Returns the user data if user was found
```

Comment:
Btw.  `@param cb - Callback` - this gives us no speciall information and additional value. Let's avoid adding comments to params. If you generate docs using ts-docs you can see them included anyway.